evap/rewards/tests/test_views.py

```diff
@@ -80,10 +80,20 @@ def test_redeem_points_for_expired_event(self):
         self.assertEqual(5, reward_points_of_user(self.student))
 
     def test_invalid_post_parameters(self):
-        self.app.post(self.url, params={"points-asd": 2}, user=self.student, status=400)
-        self.app.post(self.url, params={"points-": 2}, user=self.student, status=400)
-        self.app.post(self.url, params={f"points-{self.event1.pk}": ""}, user=self.student, status=400)
-        self.app.post(self.url, params={f"points-{self.event1.pk}": "asd"}, user=self.student, status=400)
+        point_params = {
+            "left-points": reward_points_of_user(self.student),
+            "redeemed-points": redeemed_points_of_user(self.student),
```

Comment:
Why aren't the two just called `previous_reward_points` and `previous_redeemed_points`?